test/blackbox-tests/test-cases/dune-cache/size.t/run.t

```diff
@@ -0,0 +1,26 @@
+This test checks that the dune cache size command returns the correct size of
```

Comment:
Could you inline the definitions of `dune` and `dune-project` into this file? It makes it much easier to read/modify tests.